packages/edge-gateway-link/src/index.js

```diff
@@ -18,6 +18,7 @@ const router = Router()
 
 router
   .all('*', envAll)
+  .options('*', withPreflightRequest)
```

Comment:
Personally I'd call this something like `corsPrefllightOptions` since it is a handler, not a middleware (we have middelware typically prefixed with `with`). Is also consistent with HTTP method as suffix e.g. `ipfsGet`.

Reply:
Makes sense. I will update that. Thanks